drivers/staging/android/lowmemorykiller.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
This file does not belong to this commit.

Reply:
Thank you, moved to the next commit.